src/components/mySales/page/mySalesPage.tsx

```
@@ -27,11 +34,39 @@ export default function MySalesPage() {
+  });
+
+  const { data: gradeCount } = useQuery({
+    queryKey: ['cardsCount', 'grade'],
+    queryFn: () => getMyCardsCount('grade'),
+    staleTime: 5 * 1000 * 60,
   });
 
+  const formattedGradeCount = gradeCount
+    ? Object.entries(gradeCount).reduce((acc, [grade, count]) => {
+        return {
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)